manpage/ydotoold.8.scd

```diff
@@ -20,8 +20,23 @@ ydotoold \- daemon for *ydotool*(1)
 	*-P*, *--socket-perm arg* _<perms>_
 		Set socket permission.
 
+	*-o*, *--socket-own=UID:GID*
+	    Socket ownership.
+
+	*-m*, *--mouse-off*
+	    Disable mouse (EV_REL)
+
+	*-k*, *--keyboard-off*
+	    Disable keyboard (EV_KEY)
+		
+	*-T*, *--touch-on*
+        Enable touchscreen (EV_ABS)
```

Comment:
```
Error at 24:2: Tabs are required for indentation                                                                                                                                               
make[2]: *** [manpage/CMakeFiles/ydotoold.8.dir/build.make:70: manpage/CMakeFiles/ydotoold.8] Error 1                                                                                          
make[1]: *** [CMakeFiles/Makefile2:215: manpage/CMakeFiles/ydotoold.8.dir/all] Error 2                                                                                                         
```

Reply:
my bad, fixed it on #240 and updated CI so it fails next time